models/source_list.py

```diff
@@ -516,7 +524,7 @@ def load(self, filepath=None):
         if filepath is None:
             filepath = self.get_fullpath()
 
-        if self.format == 'sepnpy':
+        if self.format in ['sepnpy', 'filter']:
```

Comment:
Is there any reason to define sepnpy and filter as two different formats?  (I'll have to read below, but so far it looks like they're synonyms.)

Update: yes, I saw below.  Ignore this.